todo_app/app.py

```diff
@@ -2,8 +2,10 @@
 from todo_app.flask_config import Config
 from todo_app.data.trello_items import get_items, add_item, update_name, update_status, delete_item
 from todo_app.view_model import ViewModel
+from dotenv import load_dotenv
 
-def create_app():
+def create_app(environ=None, start_response=None):
+    load_dotenv()
```

Comment:
We generally don't want the app to do this here, as it will potentially cause issues in your unit tests. Instead, we'd rather ask the service file to load the environment variables

See [this SO thread for more details if you're interested](https://serverfault.com/questions/413397/how-to-set-environment-variable-in-systemd-service) (and note that our suggested approach is actually the slightly out of date one!)